hrms/hr/doctype/employee_checkin/employee_checkin.json

```diff
@@ -164,10 +165,17 @@
    "fieldname": "fetch_geolocation",
    "fieldtype": "Button",
    "label": "Fetch Geolocation"
+  },
+  {
+   "default": "0",
+   "fieldname": "is_invalid",
```

Comment:
Should we call it Offshift instead of invalid?
What is Invalid needs more explanation to convey that its outside the shift timings